### PR TITLE
privacytools.io sponsorships

### LINKED ISSUE #1452: [Hypothetical] Sponsorship spots on privacytools.io
Please read and discuss this topic at the relevant PR: https://github.com/privacytoolsIO/privacytools.io/pull/1451

### PR DESCRIPTION
**Pull request by JonahAragon**
## The Program

This sponsorship program is designed to allow companies, organizations, and individuals partner with the privacytools.io team to support our vision of a more privacy-respecting internet and the greater online community.

In exchange for financial support, we will provide companies with the exposure and recognition that they are supporting privacy education online. We will do so by displaying their logo and link on various portions of privacytools.io as described on the sponsors page in this PR.

### Proposed Plans & Pricing

Context: The privacytools.io website receives well in excess of 250,000 pageviews monthly and is ranked highly for privacy-related searches.

#### Bronze: $250/month

- Your name and link along with a small logo or avatar on the sponsors page of this website.

#### Silver: $500/month

- Your medium-sized logo as a link at the top of our sponsors page.

#### Gold: $1000/month

- Your medium-sized logo as a link on the privacytools.io homepage and at the very top of our sponsors page.

## Website Recommendations

The sponsorship program will not impact any recommendations on the website. This will be ensured in a few ways.

1. Changes to recommendations on the site will be made by @privacytoolsIO/editorial only, a team within this organization that operates on a purely volunteer basis and will not receive direct compensation from any of the sponsoring companies.
2. All financial transactions will be transparently published at https://opencollective.com/privacytoolsio to be verified by the community to ensure no community member is receiving financial compensation for their recommendation.
3. All discussions regarding the recommendation or removal of a certain product will take place on GitHub and changes will require the authorization of two @privacytoolsIO/editorial team members. This team notably does not include myself (the primary beneficiary of most funding on this website, which is put towards server operation costs).
4. Sponsorships are not tied directly to recommendations through affiliate links. We will not receive compensation per user we refer to a service nor provide sponsors with information regarding how many people we recommend their service to, if their product is already recommended on the site.

## Financial Information

These sponsorships, like all contributions to privacytools.io, would be facilitated by the Open Collective Foundation, a registered 501(c)(3) charity. This means that all sponsorship spots would *also* be tax-deductible by US taxpayers.

This also means that we are legally not allowed to privately profit from any contributions given to us by our sponsors. The sole purpose of any funding we receive would go towards expenses that benefit our mission (to spread privacy education within our community and worldwide via outreach programs, and to host privacy-friendly services for our community and to serve as an example for self-hosters).

## Clarity

I want to be clear that sponsorships are not advertisements. Sponsors will not receive tracking information and should not compare this program to an advertising or referral channel. This program allows organizations to support our mission while linking their brand as a supporter of the cause.

I would also like to restate that the actual listings on this website will be reviewed solely by a team of editors on a volunteer basis, who will not be paid by sponsors to adjust the recommendations. Because our expenses and contributions are tracked in public on OpenCollective, this can be verified by the community.

## PR Information

**Netlify link:** https://deploy-preview-1451--privacytools-io.netlify.com/sponsors/

**Changes:** In addition to the /sponsors/ page, the donate link in the navbar has been replaced with the Sponsors page, and the donate link in the footer now links directly to OpenCollective. Crypto donations are still available at /donate/ and are also accessible via a donate link on the new /sponsors/ page.

Related: #899, #966. Closes #1452.

### CODE REVIEW

**Comment by Mikaela** [comment]
I haven't became any more comfortable with this regardless of hours passing and rereading everything more closely. Is this ethical and going to the right direction?

What if Facebook started sponsoring Privacytools.io? Would that make Facebook automatically good for privacy to a casual observer?

**Comment by JonahAragon** [comment]
> What if Facebook started sponsoring Privacytools.io? Would that make Facebook automatically good for privacy to a casual observer?

I don't see how it would, no?

**Comment by nitrohorse** [comment]
> > What if Facebook started sponsoring Privacytools.io? Would that make Facebook automatically good for privacy to a casual observer?
> 
> I don't see how it would, no?

I guess the issue mainly would be that Facebook’s logo (if they were a Gold level) would appear on the home page and top of the Sponsors page could give a mixed message to users. But to clarify, PTIO doesn’t support Sponsors but it’s possible a Sponsor’s services or products are recommended on the site.

**Comment by JonahAragon** [comment]
In this specific (unlikely) scenario all we are saying is that "Facebook is sponsoring privacytools.io", which would be an objective fact. This is kind of my answer to questions like https://github.com/privacytoolsIO/privacytools.io/issues/1409#issuecomment-545131416 too. From a transparency perspective this makes _more_ sense than accepting contributions via PayPal and etc. where this information is hidden from the community as we have been in the past.

**Comment by Mikaela** [comment]
Comments from Linda

> https://github.com/privacytoolsIO/privacytools.io/pull/1451/commits/fbf4d442b695f67ff1d458879aeb678e0023a725
> Jonah: Marketing done wrong. People are unlikely to respond to "we want to improve ourselves" or "we want to hear from you". The first time I heard about this failing was from my friend few months ago.
> I read up more on marketing and, like my friend said, response rate increased dramatically after telling the other party how they can become better with this X thing.
> Companies don't sell products. They sell things to make people better versions of themselves.
> The two paragraphs on sponsors.html are fine.

**Comment by davegson** [comment]
@JonahAragon 

> In this ... scenario all we are saying is that "Facebook is sponsoring privacytools.io", which would be an objective fact. ... From a transparency perspective this makes more sense than accepting contributions via PayPal and etc. where this information is hidden from the community as we have been in the past.

I do agree that this is *much better* than funding your venture in the dark (paypal and such) - but I feel you can and should decide to take things one step further. As @nitrohorse mentioned, it would be very awkward having Facebook up on the frontpage of PTIO.

So why not define to exclude companies/sponsors who are obviously misaligned with your core values and mission? You could even switch the meaning around and only include companies/sponsors who are aligned with your values. 
Both options would be a big improvement, ruling out Facebook immediately.

For the details, it would be hard to draw and define a *precise* line, but that is a different topic and totally up to the PTIO team. 😉

**Comment by davegson** [comment]
I really like the clarification! Draws a clear line to rule out shady actors but still gives space for you as a team to validate each request on a case by case basis 👍👍👍

Two sentences I feel you can improve:

-----------------

> Strong encryption is the only way to prevent governments from snooping in on our communications.

It's odd to single out governments here. There are many different types of malicious entities ready to spoof user data. I'd rephrase it to something more generic like: "to prevent malicious actors from..."

-----------------

> We reserve the right to approve or deny all sponsorships from receiving the benefits detailed below if we believe that your organization or product is not aligned with our key values.

I think there is a logical error in here, first I do not know if you can say: "approve/deny a sponsorship" Would it not rather be that you approve/deny a sponsorship request (or inquiry) [I'm *not* fluent]

Also, I'd split the sentence into two so the approve/deny part is in one sentence and the reason for denying in the latter. Otherwise the misalignment seems to be both the reason for the accept & deny.

Something like:

We reserve the right to approve or deny all sponsorship inquieries from receiving the benefits detailed below. We will deny any incoming requests if we believe the organization or product is not aligned with our key values.

Make it your own 😉
